tests/flow/test_index_updates.py

```diff
@@ -176,3 +190,30 @@ def test06_remove_indexed_prop(self):
         expected_result = []
         self.env.assertEquals(result.result_set, expected_result)
 
+    def populate_graph_many_nodes(self, graph):
+        global node_ctr
+        for i in range(10000):
+            node = Node("person", properties = {'age': i})
+            redis_graph.add_node(node)
+        redis_graph.commit()
```

Comment:
This is extremely expensive compared to:
`UNWIND range(0, 10000) AS x CREATE (:person {age: x})`